ox_profile/core/launchers.py

```diff
@@ -124,6 +124,7 @@ def __init__(self, sampler=None, stop_flag=None, interval=.001,
         self.unpaused = threading.Event()
         self.pause()
         threading.Thread.__init__(self, *args, **kwargs)
+        self.name = "ox_profiler_SimpleLauncher_Thread"
```

Comment:
Do you use `self.name` somewhere? Fine with me to have it but didn't see what this affects.

Reply:
I don't use it directly, but this name will be visible in case of debugging multithreaded application with ox_profile attached to it. By default threads are named as Thread-1, Thread-2 .. etc. And it may be difficult to understand where are your application's threads and where are the third party ones.

Reply:
Ah. That's interesting. I didn't know that. Thanks.